frontend/src/global_styles/content/_datepicker.sass

```diff
@@ -63,8 +63,8 @@ $datepicker--border-radius: 5px
         border-radius: $datepicker--border-radius
         box-shadow: none !important
 
-        &:nth-child(7n+7),
-        &:nth-child(7n+6)
+        &.flatpickr-disabled,
+        &.flatpickr-greyed-out
```

Comment:
The name `greyed-out` is not optimal imho as you don't only grey out but do other things as well. Currently it is only the border-radius, but in the long run things like `pointer-events` or `cursor` might be added. 
If we don't distinguish between "disabled" and "weekend" anyways, we can also directly use the `flatpickr-disabled` class. If we want a differentiation, something like `flatpickr-non-working-day` is probably better.

Reply:
We can't use `flatpickr-disabled` for now, as it blocks cursor events and prevents any interaction with it. That's why I introduced this temporary class which only exists while we're not blocking non-working days. I can live with `flatpickr-weekend` or `-non-working-day`